wled00/json.cpp

```diff
@@ -818,6 +820,7 @@ void serializeInfo(JsonObject root)
   #endif
   #ifndef WLED_DISABLE_OTA
   os += 0x01;
+  root[F("otaLocked")] = otaLock;
```

Comment:
IMO the if enclosing is enough (and its bitfield of options).
Adding another key seems wasteful.

Reply:
As the docs do not explain how you read this info out of that field, can you at least explain here how to read that value